package.json

```diff
@@ -35,7 +36,7 @@
   },
   "homepage": "https://github.com/square/web-sdk#readme",
   "engines": {
-    "node": ">12.*"
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Comment:
I got this range from [this discussion](https://github.com/sindresorhus/meta/discussions/15#discussioncomment-688195). seems ">=12.20" would also work. I don't have strong opinions